pkg/cmd/init.go

```diff
@@ -214,7 +214,7 @@ func certSetup(cfg *config.MicroshiftConfig) (*certchains.CertificateChains, err
 					ValidityDays: cryptomaterial.ShortLivedCertificateValidityDays,
 				},
 				Hostnames: []string{
-					"router-default.apps." + cfg.Cluster.Domain,
+					"*.apps.microshift." + cfg.BaseDomain, // wildcard for any additional auto-generated domains
```

Comment:
Is this change from `router-default` to `*` wildcard intentional?

Reply:
Yes, see my comments; it is what OCP does. Providing a wildcard domain in this certificate.

In this way both router-default and any other router generated domains will be covered by this serving cert.

Reply:
It is true that may be it does not exactly belong in this PR, but I was trying to align with what OCP did regarding the basedomain/clustername on the certs, and that's what I found.

```
[majopela@rhel9 microshift]$ openssl s_client -connect oauth-openshift.apps.ci-ln-n3z62kb-76ef8.aws-2.ci.openshift.org:443 </dev/null 2>/dev/null | openssl x509 -noout -text | grep DNS
DNS:*.apps.ci-ln-n3z62kb-76ef8.aws-2.ci.openshift.org
```

Reply:
Thanks for investigating this. Makes sense to me then,